README.md

```diff
@@ -837,7 +837,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 **[`^        back to top        ^`](#)**
 
-  * [Chevereto 2.X](https://github.com/Chevereto/Chevereto-2.X) - Open source one-click image hosting script. ([Source Code](https://github.com/Chevereto/Chevereto-2.X)) `MIT` `PHP`
+  * [Chevereto Free](https://github.com/Chevereto/Chevereto-Free) - A powerful and fast image hosting script that allows you to create your very own full featured image hosting website in just minutes. ([Source Code](https://github.com/Chevereto/Chevereto-Free)) `AGPLv3` `PHP`
```

Comment:
That's not what I meant -- You have the links to the GitHub repos twice in each item. Once as the link for the name and once in the Source Code.

Having both entries was fine, having the links twice in each item is what needs correcting.